lib/pdk/util/version.rb

```diff
@@ -0,0 +1,36 @@
+require 'pdk/version'
+require 'pdk/cli/exec'
+
+module PDK
+  module Util
+    module Version
+
+      def self.version_string
+        "#{PDK::VERSION} #{pdk_ref}".strip
+      end
+    
+      def self.pdk_ref
+        ref = pkg_sha || git_ref
```

Comment:
Debating flipping this prioritization. I think if someone installs the official pdk package, but modifies the underlying code (with an initialized git repo), we should be aware of that.